src/status_im/ui/screens/home/filter/views.cljs

```diff
@@ -0,0 +1,140 @@
+(ns status-im.ui.screens.home.filter.views
+  (:require [status-im.ui.components.list.views :as list]
+            [status-im.ui.screens.home.styles :as styles]
+            [status-im.ui.components.react :as react]
+            [status-im.i18n :as i18n]
+            [status-im.ui.components.colors :as colors]
+            [status-im.ui.screens.home.views.inner-item :as inner-item]
+            [status-im.utils.utils :as utils]
+            [status-im.ui.components.animation :as animation]
+            [reagent.core :as reagent]
+            [re-frame.core :as re-frame]
+            [status-im.ui.components.icons.vector-icons :as icons]))
+
+(defn search-input [_ {:keys [on-cancel on-focus on-change]}]
```

Comment:
Are those 2 files new code?

Reply:
no just moved filter in the new namespace